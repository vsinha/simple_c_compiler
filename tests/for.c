
long i;

void main()
{
	for (i=0; i<15; i= i+1) {
		printf("i=%d\n", i);
	}
		
	printf("OK\n");
}

