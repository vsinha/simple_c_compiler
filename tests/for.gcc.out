for.c: In function 'main':
for.c:7:3: warning: incompatible implicit declaration of built-in function 'printf' [enabled by default]
for.c:7:3: warning: format '%d' expects argument of type 'int', but argument 2 has type 'long int' [-Wformat]
for.c:10:2: warning: incompatible implicit declaration of built-in function 'printf' [enabled by default]
